signing/secp256k1/point_test.go

```diff
@@ -0,0 +1,111 @@
+package secp256k1_test
+
+import (
+	"testing"
+
+	crypto "github.com/ElrondNetwork/elrond-go-crypto"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing/secp256k1"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPoint_Equal(t *testing.T) {
```

Comment:
test also the other methods that return error? Valid also for the Scalar implementation

Reply:
done